src/views/Conditions/Conditions.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-curly-brace-presence */
+import React from 'react';
+import { Container, Box } from '@material-ui/core';
+import ConditionNav from '../../components/ConditionNav';
+import ContentSection from '../../components/ContentSection';
+import { useStyles } from './Conditions.styles';
+
+export default function Conditions(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container>
+      <Box>
+        <ConditionNav />
+        <ContentSection type={'about'} />
```

Comment:
suggestion: in such cases better to create objects/arrays and map over them.
This will allow for easier extensiblity and separates contents from the view.